.github/workflows/linux_builds.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           - name: Minimal template (target=template_release, everything disabled)
             cache-name: linux-template-minimal
             target: template_release
-            sconsflags: modules_enabled_by_default=no disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no
+            sconsflags: modules_enabled_by_default=no disable_3d=yes disable_advanced_gui=yes deprecated=no tablet=no minizip=no
```

Comment:
It's a Linux template build, but the flag is only effective on Windows, so it won't have nay effect.